### PR TITLE
Announce Jansa Beta and update pre-release

### PR DESCRIPTION
**Pull request by chessbyte**

### CODE REVIEW

**Comment by h-kataria** [comment]
@chessbyte this is missing https://github.com/ManageIQ/manageiq/issues/20162

**Comment by chessbyte** [comment]
@h-kataria it is the second link on line 23

File: site/_posts/2020-06-02-Announcing-Jansa-Beta.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Improvements to Chargeback [API](https://github.com/ManageIQ/manageiq-api/issues/781) and [Deep Linking in UI](https://github.com/ManageIQ/manageiq/issues/20162)
+- Automate Engine Performance was improved [here](https://github.com/ManageIQ/manageiq-automation_engine/issues/409) and [here](https://github.com/ManageIQ/manageiq-automation_engine/issues/410)
+- Added [Account Lockout Policy](https://github.com/ManageIQ/manageiq/issues/20043) to enforce account lockout after several failed login attempts
+
+### Providers
+- VMware inventory collection should be _much_ faster and more resource-efficient with [Streaming Refresh](https://youtu.be/qW_oQrTAGEw?t=882)
+- Kubernetes [optimized targeted refresh of inventory](https://github.com/ManageIQ/manageiq-providers-kubernetes/issues/369) by leveraging the Kubernetes Watch API to receive efficient change notifications.
+- [Added support for OpenShift 4.x](https://github.com/ManageIQ/manageiq-providers-openshift/issues/139) including the leveraging of [Kubernetes Watches](https://github.com/ManageIQ/manageiq-providers-openshift/issues/171)
```
Comment (agrare) [inline site/_posts/2020-06-02-Announcing-Jansa-Beta.md:30]: Video demo of OCPv4 support https://youtu.be/JBdDoTOl42c

File: site/_posts/2020-06-02-Announcing-Jansa-Beta.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Core
+- [OpenID-Connect or OIDC Authentication](https://github.com/ManageIQ/manageiq/issues/19867) support for both appliance and podified, for both UI and API
+- Improvements to Chargeback [API](https://github.com/ManageIQ/manageiq-api/issues/781) and [Deep Linking in UI](https://github.com/ManageIQ/manageiq/issues/20162)
+- Automate Engine Performance was improved [here](https://github.com/ManageIQ/manageiq-automation_engine/issues/409) and [here](https://github.com/ManageIQ/manageiq-automation_engine/issues/410)
+- Added [Account Lockout Policy](https://github.com/ManageIQ/manageiq/issues/20043) to enforce account lockout after several failed login attempts
+
+### Providers
+- VMware inventory collection should be _much_ faster and more resource-efficient with [Streaming Refresh](https://youtu.be/qW_oQrTAGEw?t=882)
+- Kubernetes [optimized targeted refresh of inventory](https://github.com/ManageIQ/manageiq-providers-kubernetes/issues/369) by leveraging the Kubernetes Watch API to receive efficient change notifications.
```
Comment (agrare) [inline site/_posts/2020-06-02-Announcing-Jansa-Beta.md:29]: Demo of k8s watches https://youtu.be/Sy1fs41PxL0

**Pull request merged**